packages/@uppy/compressor/types/index.d.ts

```diff
@@ -7,6 +7,12 @@ export interface CompressorOptions extends PluginOptions {
   locale?: CompressorLocale
 }
 
+export type GenericEventCallback = () => void;
+
+export interface UppyEventMap {
+  'compressor:complete': GenericEventCallback
+}
+
```

Comment:
This is not going to work without declaring the module to merge it into the other events

```suggestion
declare module '@uppy/core' {
  export interface UppyEventMap {
    'compressor:complete': () => void
  }
}

```